modules/chapter3/pages/section1.adoc

```diff
@@ -1,3 +1,68 @@
 = Section 1
 
-This is _Section 1_ of _Chapter 3_ in the *hello* quick course....
\ No newline at end of file
+== AI Pipelines
+
+Constructing an AI pipeline for a model presents significant challenges, yet segmenting and modularizing this pipeline is even more complex. A standard machine or deep learning pipeline usually starts with a sequence of preprocessing activities, then progresses through a phase of experimentation and optimization and finally does the deployment. Each stage poses distinct obstacles throughout the model development process.
+
+== Elyra
+
+Elyra provides a Pipeline Visual Editor for building AI pipelines from Jupyter notebooks, Python scripts and R scripts, simplifying the conversion of multiple notebooks or scripts files into batch jobs or workflows.
+
+Elyra extends the Jupyter Notebook UI to allow for execution of a notebook as a batch job in local or remote cloud environments. This feature leverages the AI pipelines feature and requires either a Kubeflow Pipelines or Apache Airflow deployment via a runtime configuration.
+
+image::elyra_jupyter.png[Jupyter with elyra]
+
+Elyra supports two types of components: generic components and custom components. A pipeline that utilizes only generic components is called a generic pipeline, whereas a pipeline that utilizes generic components and/or custom components is referred to as runtime-specific pipeline.
+
+The Elyra Visual Pipeline editor lets you assemble pipelines by dragging and dropping supported files onto the canvas and defining their dependencies. After you've assembled the pipeline and are ready to run it, the editor takes care of generating the Apache Airflow DAG code on the fly, eliminating the need for any coding.
+
+== Creating a Data Science Pipeline with Elyra
+
+Pipelines are created in Elyra with the Visual Pipeline Editor by:
+
+* Launch JupyterLab with the Elyra extension installed.
+* Create a new pipeline by clicking on the Elyra pipeline editor icon.
+* Add each node to the pipeline by dragging and dropping notebooks or scripts from the file browser onto the pipeline editor canvas.
+* Connect the nodes to define the flow of execution.
+* Configure each node by double-clicking on it and setting the appropriate runtime image and file dependencies.
+* Once the pipeline is complete, you can run it locally or submit it to run on Kubeflow Pipelines or Apache Airflow.
+
+== Creating a runtime configuration
```

Comment:
In our Elyra distribution, the runtime is pre-configured for submitting pipelines to Data Science Pipelines. In this introductory course, we can skip runtime configuration instructions and instead refer to the documentation.